.github/workflows/model_servers.yaml

```diff
@@ -49,7 +49,7 @@ jobs:
         id: build_image
         uses: redhat-actions/buildah-build@v2.13
         with:
-          image: ${{ env.REGISTRY }}/${{ matrix.image_name }}
+          image: ${{ env.REGISTRY }}/containers/${{ matrix.image_name }}
```

Comment:
I suggest we move our images to have the component type in them, ex: `ghcr.io/containers/llamacpp_python` --> `ghcr.io/containers/model_server/llamacpp_python`.

Reply:
That is a valid request but not supported by ghcr. Please take a look - https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#pushing-container-images

Reply:
If you want to proceed this way that is fine, in this case I was assuming the image name was `model_server/llamacpp_python` which in this case doesn't violate any rules I can see, see the DinD container for example: `ghcr.io/devcontainers/features/docker-in-docker:2.10.2`.